app-laptop/i8kutils/i8kutils-1.42.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit systemd toolchain-funcs
+
+DESCRIPTION="Dell Inspiron and Latitude utilities"
+HOMEPAGE="https://launchpad.net/i8kutils"
+SRC_URI="https://launchpad.net/i8kutils/trunk/${PV}/+download/${P/-/_}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples tk"
+
+DEPEND="tk? ( dev-lang/tk:0 )"
+RDEPEND="${DEPEND}
+         sys-power/acpi"
```

Comment:
Now you're mixing space and tab indent. In ebuilds, we always tab-indent, and never space.